tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -236,11 +237,11 @@ var activationResultCmd = &cobra.Command{
     },
 }
 //lastFlag(args) retrieves the last activation with flag -l or --last
-//Param: Brings in []strings from args
+//Param: Brings in []strings from args and the name of the function that is using it (for error feedback)
 //Return: Returns an args([]string) with last ID used or the original args with any errors
-func lastFlag(args []string) ([]string, error) {
+func lastFlag(args []string, funcName string) ([]string, error) {
```

Comment:
I don't think `funcName` is necessary here. Easier to just put a debug message in each of the if statement for the function calls.